origin-growth/migrations/20190124003026-create-growth-campaign.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const tableName = 'growth_campaign'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable(tableName, {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      name: {
+        type: Sequelize.STRING
+      },
+      rules: {
+        type: Sequelize.JSONB
+      },
+      start_date: {
+        type: Sequelize.DATE
+      },
+      end_date: {
+        type: Sequelize.DATE
+      },
+      distribution_date: {
+        type: Sequelize.DATE
+      },
+      cap: {
+        type: Sequelize.INTEGER
```

Comment:
Just to confirm we will be writing cap values in OGN tokens without the decimal precision?

Reply:
Very good point. Changing the type to bigint so that we can store values in wei.